web/packages/teleterm/src/services/logger/loggerService.ts

```diff
@@ -203,6 +210,24 @@ type FileLoggerOptions = {
   omitTimestamp?: boolean;
 };
 
+/** Does not stringify messages and logs directly using `console.x` functions. */
+function getBrowserConsoleTransport(opts: FileLoggerOptions) {
```

Comment:
Any sensitive properties are still filtered out because the filtering is done on the level of gRPC interceptors, right? I see that the regular console transport doesn't include any special filtering code either.